### PR TITLE
Add scheme for geometry checker

### PR DESCRIPTION
**Pull request by yjacolin**
Add scheme to show the different checks for the plugin

### CODE REVIEW

File: source/docs/user_manual/plugins/plugins_geometry_checker.rst [hunk 1]
```diff
@@ -67,6 +67,19 @@ The *Geometry Checker Plugin* can find the following errors:
 * Overlaps: polygon overlapping,
 * Gaps: gaps between polygons
 
+The following figure shows the different checks make by the plugin.
```
Comment (DelazJ) [inline source/docs/user_manual/plugins/plugins_geometry_checker.rst]: made

File: source/docs/user_manual/plugins/plugins_geometry_checker.rst [hunk 2]
```diff
@@ -67,6 +67,19 @@ The *Geometry Checker Plugin* can find the following errors:
...
+
+.. _figure_geometry_checker_2:
+
+.. only:: html
+
+   **Figure The Differents checks suported by the plugin**
```
Comment (DelazJ) [inline source/docs/user_manual/plugins/plugins_geometry_checker.rst]: supported

File: source/docs/user_manual/plugins/plugins_geometry_checker.rst [hunk 3]
```diff
@@ -67,6 +67,19 @@ The *Geometry Checker Plugin* can find the following errors:
...
+
+.. figure:: /static/user_manual/plugins/geometry_checker_scheme.png
+   :align: center
+
+   The Differents checks suported by the plugin
```
Comment (DelazJ) [inline source/docs/user_manual/plugins/plugins_geometry_checker.rst]: as above

**Comment by DelazJ** [comment]
Great. thank u!

File: source/docs/user_manual/plugins/plugins_geometry_checker.rst (hunk 2 above)
Comment (yjacolin) [inline source/docs/user_manual/plugins/plugins_geometry_checker.rst]: what do you mean?

Comment (DelazJ) [inline source/docs/user_manual/plugins/plugins_geometry_checker.rst]: Most of the time, this item would be like this: **Figure Geometry Checker 2** and below you have the title as you wrote... That said, i have no idea how that builds in html.
Does your choice have to do with the discussion about numbering figure title? (in which case, i wonder if  it's not the first one who needs that change).

**Pull request merged**